govcd/lbserverpool.go

```diff
@@ -0,0 +1,177 @@
+package govcd
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+// CreateLBServerPool
+func (eGW *EdgeGateway) CreateLBServerPool(lbPoolConfig *types.LBPool) (*types.LBPool, error) {
```

Comment:
The comment to this function should include some info on which elements of  `lbPoolConfig` need to be filled mandatorily

Reply:
Added.